pkg/CoreFx.Private.TestUtilities/CoreFx.Private.TestUtilities.pkgproj

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sorry for not catching this during the PR, but why did you put this pkgproj under the root pkg folder? It should go under a pkg folder in the src directory `src\CoreFx.Private.TestsUtitlies\pkg`.